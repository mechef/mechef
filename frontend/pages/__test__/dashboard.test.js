import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';

import Dashboard from '../dashboard';
jest.mock('react-geosuggest', () => '[MOCK]Geosuggest');
describe('Pages With Snapshot Testing', () => {
  it('snapshot dashboard page', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <Dashboard
        router={{
          query: {
            page: 'menu',
          },
        }}
        global={{
          backArrow: {
            isShow: false,
            title: '',
          },
        }}
        toggleBackArrow$={() => {}}
      />,
    );
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
