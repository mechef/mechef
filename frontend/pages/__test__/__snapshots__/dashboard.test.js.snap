// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages With Snapshot Testing snapshot dashboard page 1`] = `
<Translate(Dashboard)
  global={
    Object {
      "backArrow": Object {
        "isShow": false,
        "title": "",
      },
    }
  }
  showSpinner$={[Function]}
  toggleBackArrow$={[Function]}
  url={
    Object {
      "query": Object {
        "page": "menu",
      },
    }
  }
/>
`;
